scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/job/factories/StaxJobFactory.java

```diff
@@ -102,6 +103,10 @@
 
     private static final String FILE_ENCODING = PASchedulerProperties.FILE_ENCODING.getValueAsString();
 
+    private static final String PA_CATALOG_REST_URL = PASchedulerProperties.CATALOG_REST_URL.getValueAsString();
```

Comment:
I think this static initialization will create issues as the scheduler & catalog rest are not initialized on startup.

Can you do a lazy initialization instead ?